charts/osschallenge/templates/secret-backend.yaml

```diff
@@ -0,0 +1,17 @@
+apiVersion: v1
+kind: Secret
+metadata:
+  name: {{ include "osschallenge.fullname" . }}
+  labels:
+    {{- include "osschallenge.labels" . | nindent 4 }}
+type: Opaque
+data:
+  {{ if .Values.osschallenge.service.secret }}
+  DJANGO_SECRET_KEY: {{ .Values.osschallenge.settings.secret | b64enc | quote }}
+  {{ else }}
+  DJANGO_SECRET_KEY: {{ randAlphaNum 32 | b64enc | quote }}
+  {{ end }}
+  EMAIL_HOST: {{ .Values.osschallenge.settings.emailHost | quote }}
+  EMAIL_HOST_PORT: {{ .Values.osschallenge.settings.emailPort | quote }}
+  EMAIL_HOST_USER: {{ .Values.osschallenge.settings.emailUser | quote }}
```

Comment:
These belong into the ConfigMap and not the Secret... Only secrets should be stored in Secret resources.